### PR TITLE
Update Summary

### PR DESCRIPTION
**Pull request by certara-yaroslavivanov**

### CODE REVIEW

File: Certara.Nca.Ui/ClientApp/src/app/projects/summary/models/summary.models.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+interface SummaryItemContent {
```
Comment (ankitsinghcertara) [inline Certara.Nca.Ui/ClientApp/src/app/projects/summary/models/summary.models.ts]: interface name starts with I. Why is export done like this instead direct export in front of interface keyword?

Comment (certara-yaroslavivanov) [reply Certara.Nca.Ui/ClientApp/src/app/projects/summary/models/summary.models.ts]: Fixed

Comment (ankitsinghcertara) [reply Certara.Nca.Ui/ClientApp/src/app/projects/summary/models/summary.models.ts]: Is this interface used anywhere? Did you have to update that reference too?

Comment (certara-yaroslavivanov) [reply Certara.Nca.Ui/ClientApp/src/app/projects/summary/models/summary.models.ts]: Not for now. Will be used in the PR for WW-474

**Pull request merged**